app/assets/javascripts/controllers/live_metrics/ad_hoc_metrics_controller.js

```diff
@@ -110,6 +112,26 @@ ManageIQ.angular.app.controller('adHocMetricsController', ['$http', '$window', '
       filterChange(null, true);
     };
 
+    var setAppliedFilters = function() {
+      var tags = dash.params.get("tags");
+
+      // if user did not send any tags, just exit
+      if (!tags) return;
```

Comment:
I think it is better to check:
`if (tags == null)` 

(which is equivalent to: `if (tags === undefined || tags === null)`)